### ISSUE TITLE
Network hash in progress chart not correct

### ISSUE THREAD

**Issue by erbsenzaehler** [glinscott/leela-chess#193]
The hash on the Networks page does not fit to the corresponding entry on the progress chart. It is off by one.

See:

![2018-03-27-112343_1920x1080_scrot](https://user-images.githubusercontent.com/6897022/37959343-362f15c0-31b3-11e8-8c05-e71616f18bf9.png)

**Comment by jkiliani** [comment]
Good catch, and I think I just figured out what's the problem here: The progress chart tooltip shows the hash from the "current ID" of the corresponding match, not the "candidate ID" as it should. As a consequence, all tooltip hashes are wrong, and for multiple failed nets in sequence, the shown hashes are identical.

**Comment by luigio** [comment]
It's quite confusing.
